cluster-controller/src/main/java/io/enmasse/barnabas/controller/cluster/resources/Cluster.java

```diff
@@ -1,5 +1,5 @@
 package io.enmasse.barnabas.controller.cluster.resources;
 
-public interface Resource {
+public interface Cluster {
```

Comment:
Do we still need this? When the Cluster classes container the logic, this interface contained the methods such as `create`, `update`, `delete` etc. But IO'm not sure we still need this interface. Do we use it anywhere?

Reply:
I don't think we need this. We can get rid of it.